upgrade_requirements.ps1

```diff
@@ -0,0 +1 @@
+spetlr-freeze-req install_requirements.txt --cfg
```

Comment:
What does this do? Would be nice with a docstring.

Reply:
added